docs/editor/voice.md

```diff
@@ -0,0 +1,59 @@
+---
+Order:
+Area: editor
+TOCTitle: Voice interactions
+ContentId: e3bf9098-7b2f-4b23-9e0f-3d2094bad80a
+PageTitle: Using Voice in Visual Studio Code
+DateApproved:
+MetaDescription: Visual Studio Code voice accessibility features. Learn here about the various ways VS Code can be used with voice.
+---
+# Voice Support
+
+Microsoft publishes the [VS Code Speech](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-speech) extension to enable various features through the power of only using your voice. Once installed, the extension enables you to dictate into the editor, or to verbally interact with [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat).
```

Comment:
"Through the power of only using your voice" - can we tighten this up?
For example "Through the power of your voice"
I mean you can use this feature in combination with the keyboard/mouse.